taca/analysis/analysis_nanopore.py

```diff
@@ -35,79 +36,144 @@ def find_runs_to_process():
                     send_mail(email_subject, email_message, email_recipients)
+
+                    if is_not_transferred(run_id, transfer_log):
+                        if transfer_run(run_dir):
+                            update_transfer_log(run_id, transfer_log)
+                            logger.info('Run {} has been synced to the analysis cluster.'.format(run_id))
+                            archive_run(run_dir)
+                            logger.info('Run {} is finished and has been archived. Notifying operator.'.format(run_id))
+                            email_subject = ('Run successfully processed: {}'.format(os.path.basename(run_id)))
+                            email_message = ('Run {} has been analysed, transferred and archived '
+                                             'successfully.').format(run_id)
+                            send_mail(email_subject, email_message, email_recipients)
+
+                        else:
+                            logger.warn('An error occurred during transfer of run {} '
+                                        'to Irma. Notifying operator.'.format(run_dir))
+                            email_subject = ('Run processed with errors: {}'.format(os.path.basename(run_id)))
+                            email_message = ('Run {} has been analysed, but an error occurred during '
+                                             'transfer.').format(run_id)
+                            send_mail(email_subject, email_message, email_recipients)
+
+                    else:
+                        logger.warn('The following run has already been transferred, '
+                                    'skipping: {}'.format(run_dir))
+
                 else:
-                    email_subject = ('Run processed with errors: {}'.format(os.path.basename(run_dir)))
-                    email_message = 'Run {} has been analysed, but an error occurred during \
-                    transfer.'.format(run_dir)
+                    logger.warn('Anglerfish exited with a non-zero exit status for run {}. '
+                                'Notifying operator.'.format(run_dir))
+                    email_subject = ('Run processed with errors: {}'.format(os.path.basename(run_id)))
+                    email_message = ('Anglerfish exited with errors for run {}. Please '
+                                     'check the log files and restart.').format(run_id)
                     send_mail(email_subject, email_message, email_recipients)
-            else:
-                logger.warn('The following run has already been transferred, \
-                skipping: {}'.format(run_dir))
+
+            elif not qc_run:
+                if is_not_transferred(run_id, transfer_log):
+                    if transfer_run(run_dir):
+                        update_transfer_log(run_id, transfer_log)
+                        logger.info('Run {} has been synced to the analysis cluster.'.format(run_id))
```

Comment:
Just wondering which one the analysis cluster is?

Reply:
Irma 🙂